criteria/reusable-and-portable-codebases.md

```diff
@@ -12,7 +12,7 @@ order: 3
 * The roadmap SHOULD be influenced by the needs of multiple parties.
 * Code SHOULD be general purpose and SHOULD be configurable.
 * Codebases SHOULD include a [publiccode.yml](https://github.com/italia/publiccode.yml) metadata description so that they're easily discoverable.
-* Code and its documentation SHOULD not contain situation-specific information. For example, personal and organizational data as well as tokens and passwords used in the production system should never be included.
+* Code and its documentation SHOULD NOT contain situation-specific information such as hostnames.
```

Comment:
A hostname does seem like an odly specific example though to be the lone example and I quite some developers that would not really know what a hostname is, let alone managers and designers, although I would like them to make sure of this as well.

I'd say perhaps we can leave the example off until we find a more examplary example or add some more.

```suggestion
* Code and its documentation SHOULD NOT contain situation-specific information.
```